lib/features/rules_filter_creator/presentation/rules_filter_creator_controller.dart

```diff
@@ -48,21 +50,19 @@ class RulesFilterCreatorController extends BaseMailboxController {
   GetAllRulesInteractor? _getAllRulesInteractor;
 
   final errorRuleName = Rxn<String>();
-  final errorRuleConditionValue = Rxn<String>();
+  final listErrorRuleConditionValue = RxList<String>();
   final errorRuleActionValue = Rxn<String>();
-  final ruleConditionFieldSelected = Rxn<rule_condition.Field>();
-  final ruleConditionComparatorSelected = Rxn<rule_condition.Comparator>();
   final emailRuleFilterActionSelected = Rxn<EmailRuleFilterAction>();
   final mailboxSelected = Rxn<PresentationMailbox>();
   final actionType = CreatorActionType.create.obs;
+  final listRuleCondition = RxList<RuleCondition>();
 
   final TextEditingController inputRuleNameController = TextEditingController();
-  final TextEditingController inputConditionValueController = TextEditingController();
+  final List<TextEditingController> listInputConditionValueController = [];
   final FocusNode inputRuleNameFocusNode = FocusNode();
-  final FocusNode inputRuleConditionFocusNode = FocusNode();
+  final List<FocusNode> listInputRuleConditionFocusNode = [];
```

Comment:
listErrorRuleConditionValue, listInputConditionValueController, listErrorRuleConditionValue are same length,
so I think better is combine it into a class, then define a List for that class instance only